cli.js

```diff
@@ -1,4 +1,6 @@
-import generate from './index.js';
-import argv from './lib/cliArgs.js';
+const imports = require('esm')(module);
+
+const generate = imports('./index.js').default;
+const argv = imports('./lib/cliArgs.js').default;
```

Comment:
Can these be changed to normal `import`, or does it have to use `imports`?

Reply:
Would be a duplicate declaration, and afaik not serve any purpose since this entry is only doing one thing. Surface area for changing back to completely pure es modules in the future is pretty minimal though!